frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -314,6 +315,23 @@ export const applySquareRootScaling = (value: RowValue): RowValue => {
   return value;
 };
 
+function filterNullDimensionValues(dataset: ChartDataset) {
+  // TODO show warning message
```

Comment:
This is part of another issue

Reply:
Can you please update `null-category-value-formatting.json` visualization_settings.graph.x_axis.scale => "ordinal"?

Reply:
Yup let me do that